### PR TITLE
Ros2 rtd plugin

### PR DESCRIPTION
**Pull request by wep21**
### How to test
```
rviz2 -d (your rviz config)
ros2 topic pub /control/turn_signal_cmd autoware_vehicle_msgs/msg/TurnSignal
ros2 topic pub /vehicle/status/steering autoware_vehicle_msgs/msg/Steering
ros2 topic pub /vehicle/status/twist geometry_msgs/msg/TwistStamped
```
Screenshot from 2021-03-18 13-45-55

### CODE REVIEW

**Review by isamu-takagi — APPROVED** [review]
LGTM

File: common/util/rviz_plugins/autoware_vehicle_rviz_plugin/src/tools/jsk_overlay_utils.cpp [hunk 1]
```diff
@@ -76,11 +76,9 @@ QImage ScopedPixelBuffer::getQImage(OverlayObject & overlay, QColor & bg_color)
   return getQImage(overlay.getTextureWidth(), overlay.getTextureHeight(), bg_color);
 }
 
-OverlayObject::OverlayObject(
-  Ogre::SceneManager * manager, rclcpp::Logger logger, const std::string & name)
-: name_(name), logger_(logger)
+OverlayObject::OverlayObject(const std::string & name)
+: name_(name)
 {
-  rviz_rendering::RenderSystem::get()->prepareOverlays(manager);
```
Comment (kenji-miyake) [inline common/util/rviz_plugins/autoware_vehicle_rviz_plugin/src/tools/jsk_overlay_utils.cpp:83]: [Question] Why `manager` dependency was removed? Is that necessary or something like refactoring?

File: common/util/rviz_plugins/autoware_vehicle_rviz_plugin/src/tools/steering_angle.cpp [hunk 2]
```diff
@@ -162,8 +159,7 @@ void SteeringAngleDisplay::processMessage(
 
   QMatrix rotation_matrix;
   rotation_matrix.rotate(
-    static_cast<qreal>(
-      std::round(property_handle_angle_scale_->getFloat() * (msg_ptr->data / M_PI) * -180.0)));
+    std::round(property_handle_angle_scale_->getFloat() * (msg_ptr->data / M_PI) * -180.0));
```
Comment (kenji-miyake) [inline common/util/rviz_plugins/autoware_vehicle_rviz_plugin/src/tools/steering_angle.cpp:162]: [Question] Why `static_cast` was removed? A lint issue?

File: common/util/rviz_plugins/autoware_vehicle_rviz_plugin/src/tools/jsk_overlay_utils.cpp [hunk 3]
```diff
@@ -127,11 +125,15 @@ void OverlayObject::updateTextureSize(unsigned int width, unsigned int height)
...
+    RCLCPP_WARN(
+      rclcpp::get_logger("OverlayObject"),
+      "width=0 is specified as texture size");
     width = 1;
   }
   if (height == 0) {
-    RCLCPP_WARN(logger_, "height=0 is specified as texture size");
+    RCLCPP_WARN(
+      rclcpp::get_logger("OverlayObject"),
+      "height=0 is specified as texture size");
```
Comment (kenji-miyake) [inline common/util/rviz_plugins/autoware_vehicle_rviz_plugin/src/tools/jsk_overlay_utils.cpp:136]: [Question] Why `logger` dependency was removed?

**Review by kenji-miyake — COMMENTED** [review]
@wep21 LGTM and I've confirmed this resolves the rviz-crush problem, but I have some questions so let me confirm, please.

File: common/util/rviz_plugins/autoware_vehicle_rviz_plugin/src/tools/jsk_overlay_utils.cpp (hunk 1 above)
Comment (wep21) [reply common/util/rviz_plugins/autoware_vehicle_rviz_plugin/src/tools/jsk_overlay_utils.cpp:83]: I thought this line was necessary for overlay visualization when porting this plugin, but I think it's just my misunderstanding or some rviz updates make it unnecessary.

File: common/util/rviz_plugins/autoware_vehicle_rviz_plugin/src/tools/steering_angle.cpp (hunk 2 above)
Comment (wep21) [reply common/util/rviz_plugins/autoware_vehicle_rviz_plugin/src/tools/steering_angle.cpp:162]: I thought this was one cause of rviz crush problem, but I think this is also my misunderstanding.

File: common/util/rviz_plugins/autoware_vehicle_rviz_plugin/src/tools/jsk_overlay_utils.cpp (hunk 3 above)
Comment (wep21) [reply common/util/rviz_plugins/autoware_vehicle_rviz_plugin/src/tools/jsk_overlay_utils.cpp:136]: On second thoughts, this is more simple than passing logger to class.

Comment (kenji-miyake) [reply common/util/rviz_plugins/autoware_vehicle_rviz_plugin/src/tools/jsk_overlay_utils.cpp:136]: @wep21 So do you mean that
- in https://github.com/tier4/AutowareArchitectureProposal.iv/pull/246 you fixed to inject `logger_`
- but now you change your mind it was not necessary
Is this right?

**Review by kenji-miyake — APPROVED** [review]
Anyway the bug was fixed, so I'll merge this.

**Pull request merged**